components/metris/internal/provider/azure/instance_test.go

```diff
@@ -188,7 +188,7 @@ func TestInstance_getComputeMetrics(t *testing.T) {
 			want:   metricsNoDisk,
 		},
 		{
-			name:    "with last event",
+			name:    "with last eventData",
```

Comment:
Do we need this change?

Reply:
i will revert them, goland was running rogue